app/pods/node/model.js

```diff
@@ -0,0 +1,36 @@
+import DS from 'ember-data';
+
+export default DS.Model.extend({
+  /**
+   * Cluster the node belongs to.
+   * @property cluster
+   * @type DS.Model
+   */
+  cluster: DS.belongsTo('cluster', { async: true }),
+
+  /**
+   * Whether or not the node is available when pinged.
+   *
+   * @property available
+   * @type Boolean
+   */
+  available: DS.attr('boolean', { default: false }),
+
+  /**
+   * All of the nodes configuration settings, stored as an array of key-value objects.
+   *  ex. [{key: 'anti_entropy', value: 'active'}, {key: 'anti_entropy.bloomfilter', value: 'on'}...]
+   *
+   * @method config
+   * @return {Array<Object<Config>>}
+   */
+  config: DS.attr(),
+
+  /**
+   * All of the nodes statistics, stored as an array of key-value objects.
+   *  ex. [{key: 'asn1_version', value: '2.0.3'}, {key: 'basho_stats_version', value: '1.0.3'}...]
+   *
+   * @method stats
+   * @return {Array<Object<Stats>>}
+   */
+  stats: DS.attr()
```

Comment:
Same thing with the stats. Plain JS object (hashmap), turn it into an array for display only in a separate function.